setup.sh

```diff
@@ -1,2 +1,9 @@
 #!/bin/bash
-sudo service codedeploy-agent status
\ No newline at end of file
+sudo service codedeploy-agent status
+chmod +x /home/ec2-user/nina-indexer/run.sh
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
+
+cd /home/ec2-user/nina-indexer
+npm install
```

Comment:
can we run `yarn` here instead of `npm install` since I believe we use yarn elsewhere?